chia/daemon/server.py

```diff
@@ -144,6 +145,18 @@ def _get_keys_by_fingerprints(fingerprints: Optional[List[uint32]]) -> Tuple[Lis
     return keys, missing_fingerprints
 
 
+@dataclasses.dataclass(frozen=True)
+class StatusMessage:
+    service: str
+    command: str
+    destination: str
+    origin: str
```

Comment:
Should these be more clearly constrained than just `str`?  An enumeration?  I know the remote entities service id is just any string they want but I think some of these are internal here.

Reply:
command could be constrained to an enum as there are a known fixed number of supported commands. it's currently defined in a dict in `get_command_mapping`

Reply:
I think it's just easier to leave this as is.